src/main/java/com/spartronics4915/frc2020/subsystems/Indexer.java

```diff
@@ -132,7 +132,7 @@ public void rotateN(double N)
         if (N != 0)
         {
             double deltaPosition = 0.25 * N; // Cast N to double and convert to rotations
-            mTargetPosition += deltaPosition;
+            mTargetPosition = deltaPosition + mIndexerMotor.getEncoder().getPosition();
```

Comment:
This change is not desirable. We want to keep track of target positions. If we used this PR then the target position would drift.